Prosjektrapport-V2.md

```diff
@@ -81,10 +81,22 @@ Det er viktig å dokumentere ikke bare alle hensyn som tas, men også hvilke hen
 
 ### Aktører og UseCase(UC)
 
-### UC-diagram
+Definisjon: En UseCase er en beskrivelse av en sekvens av handlinger som utføres av brukerne innenfor er system, som gir observerbar verdi til en aktør. Aktøren kan være en person eller et annet system. UseCase fokuserer på hva systemet gjør og ikke hvordan det gjøres. 
+    
+Formål: Hovedformålet med UseCase er å fange opp funksjonelle krav ved å beskrive hvordan systemet skal oppføre seg i forskjellige sitasjoner. 
+    
+    
+
 
+### UC-diagram
+Her skal det settes in UC diagram
 ### UC-beskrivelser
+Use Case beskrivelser
+Tabell UC1 til UC4 beskriver fire Use Cases som er tilknyttet applikasjonen Hytte App. 
+Tabellene viser Use Cases, hvem som er aktør, pre og post betingelser, hovedflyt og alternativ flyt hvis det skjer feil under prosessen.
+
 
+Her skal det settes inn tabeller
```

Comment:
Hvis det er flere tabeller, så kan hver tabell ha en egen tekst så det blir enklere å se akkurat hvor man skal putte de når man setter sammen det siste dokumentet.

For eksempel: Tabell: UC1

Hvis ikke kan det stå Tabell: UC1 til UC4.